core/digital-twin-lib/src/DAppTwin.ts

```diff
@@ -158,6 +164,15 @@ class DAppTwin extends DigitalTwin {
     });
   }
 
+  async ensureSharingContext(): Promise<void> {
```

Comment:
doesn't check any existing properties so I would it name it better setContactsAndBmailContent or something like this ...